kafka/consumer.go

```diff
@@ -197,8 +239,15 @@ func (c *Consumer) Close() (err error) {
 //                                        If set to true the app must handle the AssignedPartitions and
 //                                        RevokedPartitions events and call Assign() and Unassign()
 //                                        respectively.
-//   go.events.channel.enable (bool, false) - Enable the Events channel. Messages and events will be pushed on the Events channel and the Poll() interface will be disabled.
-//                                        (Experimental)
+//   go.events.channel.enable (bool, false) - Enable the Events channel. Messages and events will be pushed on the Events channel and the Poll() interface will be disabled. (Experimental)
+//   go.events.channel.size (int, 1000) - Events channel size
+//
+// WARNING: Due to the buffering nature of channels (and queues in general) the
```

Comment:
I dont know..
If we dont provide this interface people will end up wrapping chan <- Consume() in a go routine anyway, this at least documents the undesired behaviour.